src/remoteConnector.ts

```diff
@@ -707,7 +708,7 @@ export default class RemoteConnector extends Disposable {
 				sshDestination = destination;
 
 				if (password) {
-					await this.showSSHPasswordModal(password, params.gitpodHost);
+					await this.showSSHPasswordModal(password, params.gitpodHost, params);
 				}
 
 				this.telemetry.sendRawTelemetryEvent('vscode_desktop_ssh', { kind: 'gateway', status: 'connected', ...params, gitpodVersion: gitpodVersion.raw });
```

Comment:
If we need to know if user selected `password` then just add another property here

Reply:
is this (add `password` property) good for you? @akosyakov

Reply:
We only know user copied the `password` but don't know if user paste and connect it. So maybe this tracking event's `status=connected` is not correct?

Reply:
We are going to remove errors from vscode_desktop_ssh eventually and log it to GCP. We should not rely on it. We should have one event which shows us a usage of this notification, i.e. captures all exits.